pkgs/applications/video/mirakurun/generate.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nodePackages.node2nix
+
+# development packages are needed to build EPGStation
+node2nix -8 -i package.json \
```

Comment:
Don't use node2nix in nixpkgs, look up in the nixpkgs manual for node packages are packaged.